types/components/buttons.d.ts

```diff
@@ -153,3 +158,7 @@ export interface PayPalButtonsComponent {
     isEligible: () => boolean;
     render: (container: HTMLElement | string) => Promise<void>;
 }
+
+export type Mio = {
```

Comment:
I don't think I've seen `Mio` before. What is it?

Reply:
yeah, sorry I was making I test, forget to remove it... 😢